package.json

```diff
@@ -22,7 +22,20 @@
   "license": "Apache-2.0",
   "dependencies": {
     "pbf": "^3.2.1",
+    "prettier": "^2.0.5",
     "puppeteer": "^4.0.1",
     "queue": "^6.0.1"
+  },
+  "devDependencies": {
+    "eslint": "^7.4.0",
+    "eslint-config-airbnb": "^18.2.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-jsx-a11y": "^6.3.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.3",
+    "eslint-plugin-react-hooks": "^4.0.0"
```

Comment:
are all of these plugins required due to peer deps or something? If not, I'd suggest limited to only the ones needed as to not be misleading that they are in use

Reply:
At least I did not install them myself. I only installed prettier, eslint, eslint-config-prettier and eslint-plugin-prettier. The rest got installed automatically. I don't know why react is in there though or how to disable it.